build/__tools__/run-tests.sh

```diff
@@ -351,6 +351,25 @@ done
 # clean up the pdo files that are created by the shell
 rm -f ${SCRIPTDIR}/red_issuer.pdo ${SCRIPTDIR}/red_type.pdo ${SCRIPTDIR}/red_vetting.pdo
 
+
+cd ${SRCDIR}/build
+
+yell run tests for state replication 
+say start mock-contract test with replication 3 eservices 2 replicas needed before txn. 
+
+try pdo-test-request --ledger ${PDO_LEDGER_URL} \
+    --pservice http://localhost:7001/ http://localhost:7002 http://localhost:7003 \
+    --eservice-name e1 e2 e3 \
+    --logfile __screen__ --loglevel warn --iterations 10 \
```

Comment:
instead of worrying about the names can we just agree to use only URLs for the test scripts?

Reply:
of course, we can stick to URLs if that makes reading easy. I will make the change